composer.json

```diff
@@ -246,4 +249,4 @@
             ]
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
Can you add the test namespace to autoload-dev as well?

Reply:
Done! Could we please make sure this is included in the add component tooling?

Reply:
yep

Reply:
yep!